packages/test-workflows/src/resolvable-promise.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+
+/**
+ * Helper for creating promises which can be manually resolved
+ * Copied from workflow/common
```

Comment:
Why does it need to be copied?

Reply:
It's a limitation of the module loader, that module isn't injected into the workflow runtime.
I should probably add a comment.